RELEASE_NOTES.md

```diff
@@ -1,5 +1,31 @@
 # ChefDK 3.7 Release Notes
 
+## Chef 14.9.13
```

Comment:
Since we're releasing 14.10 today won't this next release ship with 14.10

Reply:
We'll wait.

Reply:
now included